packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx

```diff
@@ -1,96 +1,151 @@
+| dateline               | array    | `[]`                                           | no       | See [Dateline](./Dateline).                                                           |
+| eventline              | array    | `[]`                                           | no       | See [Eventline](./Eventline).                                                         |
+| dimension              | object   | `{}`                                           | no       | See [dimension](#dimension).                                                          |
+| legendPadding          | object   | `{ top: 4, bottom: 4, left: 8, right: 8 }`     | no       | Sets the legend padding and removes the carbon default margin of `8px` around legend. |
+| locale                 | object   | `LOCALE.en_US`                                 | no       | Locale object for x-Axis tick values.                                                 |
+| opaqueBackground       | false    | `false`                                        | no       | Toggle between a `transparent` or `white` background for the container.               |
+| padding                | object   | `{ top: 10, bottom: 10, left: 30, right: 50 }` | no       | See [Padding](./Padding).                                                             |
+| pan                    | object   | `{}`                                           | no       | See [Panning](./Panning).                                                             |
+| removeContainerPadding | boolean  | `false`                                        | no       | Toggle to remove the top and bottom padding of the carbon container.                  |
+| showHGrid              | boolean  | `true`                                         | no       | Toggle to show horizontal grid.                                                       |
+| showLabel              | boolean  | `true`                                         | no       | Toggle to show axes labels.                                                           |
+| showLegend             | boolean  | `true`                                         | no       | Toggle to show graph legend.                                                          |
+| showNoDataText         | boolean  | `true`                                         | no       | Toggle to show no data text.                                                          |
+| showShapes             | boolean  | `true`                                         | no       | Toggle to show shapes in the line graph.                                              |
+| showVGrid              | boolean  | `true`                                         | no       | Toggle to show vertical grid.                                                         |
+| ticksCount             | number   | `3`-`7` depending on the input range           | no       | See [ticksCount](#ticksCount).                                                        |
```

Comment:
tick Count hyperlink is working as expected.

Reply:
fixed here: 741f50f900bbdf14d28172159fa3087416114641